modules/@ergo/core/components/Layout/Footer/Footer.vue

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright © Bold Brand Commerce Sp. z o.o. All rights reserved.
+ * See LICENSE for license details.
+ */
+<template>
+    <footer
+        :class="[
+            'footer',
+            { 'space-between': spaceBetween },
+            { 'flex-end': flexEnd },
+            { 'flex-start': flexStart },
```

Comment:
maybe it is better to set one of this values as default ?

Reply:
Flex has default value in it selfs